test/DynamoCoreTests/Search/SearchDictionaryTest.cs

```diff
@@ -93,7 +93,7 @@ public void TestSearchDictionaryPerformance()
             Assert.AreEqual(results.Count(), 20);
 
             double timeLimit = 300;//ms
-            double range = 30;
+            double range = 70;
             Assert.IsTrue(Math.Abs(stopwatch.ElapsedMilliseconds - timeLimit) < range / 100 * timeLimit, $"Search time should be within a range of +/- {range}% of {timeLimit}ms but we got {stopwatch.ElapsedMilliseconds}ms");
```

Comment:
So we are okay with a 70% variation from the expected time?

Reply:
It's hard to predict the performance of all machines this test will run on including VMs that might be on shared machines themselves.

Reply:
Perhaps we could get some kind of coarse performance metrics from the VM (like vCPU count?) and have different scaling factors for different levels of expected performance? With a wide error margin we aren't getting as much value out of running performance tests.